openshift/4.0/templates/logging/README.md

```diff
@@ -1,49 +0,0 @@
-# Monitor Logs
```

Comment:
This may be contentious, but if we aren't using kibana/grafana etc I vote we remove the configs.

Reply:
I think it might be helpful to add them to a different folder so they are available for future possible use.  I realize they would exist in the git history (unless it was purged at some point), but they're much harder to look for when that occurs.